cluster/images/uxp-bootstrapper/Makefile

```diff
@@ -30,9 +30,4 @@ img.build.shared:
 	@docker buildx build $(BUILD_ARGS) \
 		--platform $(IMAGE_PLATFORMS) \
 		-t $(IMAGE) \
-		$(IMAGE_TEMP_DIR) || $(FAIL)
-
-img.promote:
-	@$(INFO) docker promote $(FROM_IMAGE) to $(TO_IMAGE)
-	@docker buildx imagetools create -t $(TO_IMAGE) $(FROM_IMAGE)
-	@$(OK) docker promote $(FROM_IMAGE) to $(TO_IMAGE)
\ No newline at end of file
+		$(IMAGE_TEMP_DIR) || $(FAIL)
```

Comment:
@nullable-eth does the build submodule handle the absence of the promote target well? I seem to remember a target needing to exist even if it just prints a message

Reply:
I guess you are right https://github.com/upbound/build/blob/559e1532cd1ec6471c295b9dfce89c38f9a4638f/makelib/imagelight.mk#L98 doesn't seem like it would like it if the target didn't exist.  Updated.